### PR TITLE
Create EventCatalog site

### PR DESCRIPTION
**Pull request by gregtyler**
```
npx @eventcatalog/create-eventcatalog@latest .
```

And then lots of rewriting the demo details. I've set up the domain and services, but only included one event for now.

One oddity is the `patch-nextjs-config` script which is needed to run the site locally, since a nested dependency tries to use `fs`.

#major

### CODE REVIEW

File: public/opengraph.png
Comment (acsauk) [inline public/opengraph.png:1]: Do we need this?

**Review by acsauk — COMMENTED** [review]
Looking really cool - one Q on a .png and then ready to merge :shipit:

File: public/opengraph.png
Comment (gregtyler) [reply public/opengraph.png:1]: Nope, removed 👍🏽

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**